packages/webpack/test/fixtures/main/webpack.config.js

```diff
@@ -33,6 +33,7 @@ function makeConfig(lmOptions = {}) {
     output: {
       filename: '[name].js',
       path: '/dist',
+      publicPath: '', // running bundles under node errors out when webpack attempts to figure out relative paths at runtime if this is not set
```

Comment:
wtf

there used to be a `node` flag for webpack (haven't needed it in years); unsure if it's applicable in this case.

Reply:
node flag is orthogonal to this. when you use features that rely on figuring out paths, webpack needs this field to exist, otherwise it fails. 
Also, we're running bundles targeting the browser in node when we test.